internal/store/common_unittest_db.go

```diff
@@ -35,21 +38,28 @@ func PrepareDBForUnitTests(log *logrus.Logger) (Store, *config.Config, string, *
 	}
 
 	err = store.InitialMigration()
-	Expect(err).ShouldNot(HaveOccurred())
+	if err != nil {
+		log.Fatalf("running initial migration: %v", err)
+	}
 
 	return store, cfg, randomDBName, db
 }
 
-func DeleteTestDB(cfg *config.Config, store Store, dbName string) {
+func DeleteTestDB(log *logrus.Logger, cfg *config.Config, store Store, dbName string) {
```

Comment:
this allows for benchmarks to utilize this function without depending on ginkgo.